.github/semantic.yml

```diff
@@ -32,6 +32,7 @@ scopes:
   - codecs # Anything related to Vector's codecs (encoding/decoding)
   - compression # Anything related to compressing data within Vector
   - config # Anything related to configuring Vector
+  - core # Anything related to core crates i.e. vector-core, core-common, etc
```

Comment:
This file is actually managed over here: https://github.com/vectordotdev/resources/blob/master/vector/github/semantic_pull_requests.tf

Reply:
I realized that when I went into the resources repo to do the label stuff. 😂 😭 

Hopefully TF doesn't care that it already exists in that state when I make the PR... 👀